scss/application.scss

```diff
@@ -1,3 +1,6 @@
+@import 'equalizer';
+@import 'styles';
+
 .title {
```

Comment:
This is now imported from `@import 'styles';`.
Delete this style from this file